src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.ts

```diff
@@ -74,7 +75,8 @@ export class CrushmapComponent implements OnInit {
   }
 
   onNodeSelected(e: NodeEvent) {
-    const { name, type, status, ...remain } = this.metadataKeyMap[e.node.id];
```

Comment:
Could we use `as number` instead of importing `lodash` just to make the compiler happy?
If yes I would prefer `as number` instead.

Reply:
Same as before, will revert it.